src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp

```diff
@@ -48,17 +56,29 @@ std::string getContextEntityTypeAttribute
   ParseData*                 parseDataP
 )
 {
-  std::string                          entityType     = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
-  DiscoverContextAvailabilityRequest*  requestP       = &parseDataP->dcar.res;
-  EntityId                             entityId(".*", entityType, "true");
+  std::string  entityType     = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
+  std::string  answer;
 
   LM_T(LmtConvenience,
```

Comment:
In `getContextEntityAttributes()` there was a similar LM_T() that you have deleted... It is a bit uncoherent to have the LM_T() is come cases and not in the others, isn't it?